src/main/java/org/alfresco/repo/content/transform/TikaPoweredContentTransformer.java

```diff
@@ -290,9 +300,27 @@ protected void transformRemote(RemoteTransformerClient remoteTransformerClient,
     {
         String transform = getTransform();
         long timeoutMs = options.getTimeoutMs();
+        String notExtractBookmarksText = null;
+
+        buildExtraOptionsforTika(options);

```

Comment:
I might be wrong, but I don't think this approach is going to work. Don't there need to be toMap and copyFrom methods in the TransformationOptions object? By the time you get here the action service will have thrown away the bits of the map you are using. 

If it does work, I would remove the "forTika" from the name as it might get used in another transformer.